data-plane/docker/Dockerfile

```diff
@@ -27,17 +27,20 @@ COPY dispatcher/pom.xml dispatcher/pom.xml
 COPY generated/pom.xml generated/pom.xml
 COPY tests/pom.xml tests/pom.xml
 COPY checkstyle/suppression.xml checkstyle/suppression.xml
+COPY ./mvnw .
```

Comment:
I think it's ok to keep using the maven from the docker image

Reply:
I think with this change we have a more homogeneous environment because before we ran tests (unit and build) with a maven-based image (I don't know which `JDK` build uses underneath), but for e2e tests, we used `adoptopenjdk`-based image. Now, we use only one image across all tests, and we can maintain only one Java version (in one place), besides regularly updating the wrapper that we should do anyway regardless of how we run tests. 

Is it ok for you to keep this change? Am I missing something?

Reply:
ok :wink: